cirq_superstaq/service.py

```diff
@@ -363,8 +363,8 @@ def aqt_compile_eca(
             object whose .circuits attribute is a list (or list of lists) of logically equivalent
                 cirq Circuit(s).
             If qtrl is installed, the object's .seq attribute is a qtrl Sequence object of the
-            pulse sequence corresponding to the cirq.Circuits and the .pulse_lists attribute is
-            the list(s) of cycles.
+                pulse sequence corresponding to the cirq.Circuits and the .pulse_lists attribute is
```

Comment:
I believe we use spaces over tabs
```suggestion
                pulse sequence corresponding to the cirq.Circuits and the .pulse_lists attribute is
```

Reply:
I don't know if github detects the change I suggested (I replaced the tab with 4 spaces). If not, you can just change it locally.

Reply:
@vtomole looks like this file uses tabs, but when Stephanie changes it to spaces, git doesn't see any diff